systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/RollingUpdateST.java

```diff
@@ -139,9 +163,36 @@ void testRecoveryDuringZookeeperRollingUpdate(ExtensionContext extensionContext)
             .withConsumerGroup(ClientUtils.generateRandomConsumerGroup())
             .build();
 
-        resourceManager.createResourceWithWait(extensionContext, clients.consumerTlsStrimzi(testStorage.getClusterName()));
+        resourceManager.createResourceWithWait(extensionContext, clients.consumerStrimzi());
         ClientUtils.waitForConsumerClientSuccess(testStorage);
 
+        // Kafka recovery
+
+        KafkaResource.replaceKafkaResourceInSpecificNamespace(testStorage.getClusterName(), k -> {
+            k.getSpec()
+                .getKafka()
+                .setResources(new ResourceRequirementsBuilder()
+                    .addToRequests("cpu", new Quantity("100000m"))
```

Comment:
adding more descriptive comment would be great ;) What is the assumption (e.g., if we set CPU to really... you know where I am going with it :)